doc/installation.rst

```diff
@@ -21,19 +21,37 @@ If you are on Debian or Ubuntu Linux we recommend you try
 
 .. _install-packages:
 
+
+Using Anaconda:
+===============
+
+On all platforms, you can use Anaconda_ to install Dipy. To do so issue the following two commands in a terminal::
+
+    conda config --add channels conda-forge
+    conda install dipy
+
+Some of the visualization methods require the VTK_ library and this can be installed separately (for the time being only on Python 2.7)::
+
+    conda install vtk
```

Comment:
And if we do ```conda install dipy``` first do we need to install vtk too? Or it get's directly installed?